Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16
+ 
+WORKDIR /usr/src/app
+ 
+COPY package.json .
+COPY yarn.lock .
```

Comment:
I'd move this line below the yarn install, this file might not exist yet, right?

Reply:
This file is committed to the repository already, so shouldn't be a problem.
And if we wouldn't include it, then some of the dependencies may be installed with different versions from the ones we have tested before. So it's preferred to have it here.